index.Rmd

```diff
@@ -1,105 +1,299 @@
-library(leaflet)
-library(kableExtra)
-knitr::opts_chunk$set(cache=TRUE)  # cache the results for quick compiling
+
+```{r}
+#Here is an example for Landsat data of different size
+res(Band1) # Size find is 10x10
+res(Band2) # Size find is 10x10
+res(Band3) # Size find is 10x10
+res(Band4) # Size find is 20x20
+res(Band5) # Size find is 20x20
+res(Band6) # Size find is 20x20
+res(Band7) # Size find is 10x10
+
+#Here is the code use to find size of Martian data
+res(band1) # Size find is 10x10
+res(band2) # Size find is 10x10
+res(band3) # Size find is 10x10
+res(band4) # Size find is 20x20
+res(band5) # Size find is 20x20
+res(band6) # Size find is 20x20
+res(band7) # Size find is 10x10
+res(band8) # Size find is 10x10
+res(band9) # Size find is 10x10
 ```
 
-## Download and clean all required data
+I resize now some image to make sure all image have the same size before adding them together.
+From the results above, I then multiply each image (band1, band2, band3, and band7) by 2 to have 20x20 for Earth data. However, for Martian data, we have the same size so we will just add them all together later without resizing any image.
+
+
 ```{r}
-n=20
-data=data.frame(x=runif(n,-180,180),
-                y=runif(n,-60,60),
-                size = runif(n, 5, 20),
-                category = factor(
-                  sample(letters[1:5], n, replace = TRUE)
-                  ),
-                value = rnorm(n))
+#This is for Earth (Landsat) data to show how it works
+band1 <- aggregate(band1, fact = 2) # multiply the size of band1 by 2 to have 20x20
+band2 <- aggregate(band1, fact = 2) # multiply the size of band2 by 2 to have 20x20
+band3 <- aggregate(band1, fact = 2) # multiply the size of band3 by 2 to have 20x20
+band7 <- aggregate(band1, fact = 2) # multiply the size of band7 by 2 to have 20x20
 ```
 
-```{r, results='asis'}
-data %>% 
-  slice(1:10) %>% #show only 1:n rows
-  kable(digits=2,align="c")%>% #make table and round to two digits
-  kable_styling(bootstrap_options = 
-                  c("striped", "hover", "condensed", "responsive")) #apply other formatting
+Now I can add all bands together to have a metadata containing all informations I am looking for and which will be helpful to continue the project.
+
+For Earth data, I decided to add only band2, band3 and band7 together because those wavelength are able to reflect the distribution of water on the sites. Also I just used those bands because the code does not work when I add the remaining band to the current I already stack together.
+
+However, only the Martian data is important here because we are working on Mars. So  for Martian data, we have the same size so we will just add them all together.But I add only band4, band5, band6, band8 and band9 together because those wavelength are able to reflect the distribution of water on the sites. Also I just used those bands because the code does not work when I add the remaining band to the current I already stack together.
```

Comment:
Just pick one dataset and focus on that - no need to introduce data and then not use it.